disnake/enums.py

```diff
@@ -678,6 +678,7 @@ class GuildScheduledEventStatus(Enum):
     active = 2
     completed = 3
     canceled = 4
+    cancelled = 4
```

Comment:
```suggestion
    cancelled = canceled
```

Reply:
I think we should keep enum aliases consistent, which we are already using the style above in other locations: https://github.com/DisnakeDev/disnake/blob/v2.5.1/disnake/enums.py#L309-L318